src/Option/README.md

```diff
@@ -0,0 +1,26 @@
+# Option:
+
+Provides an Option component that is passed as a child to the ListBox component.
+
+## API
+
+### `children` (element array; required)
+`children` specifies the list of elements that will be displayed within the Option component.
```

Comment:
Can we make the type of this option be node? That way we could have options that are just strings, numbers, one element, or lists of elements.